requirements-dev.in

```diff
@@ -2,3 +2,5 @@
 build
 twine~=3.6.0
 pip-tools~=6.4.0
+mkdocs
+setuptools_scm[toml]>=6.2
```

Comment:
I don't _think_ this is necessary. Listing it in build-system.requires should be sufficent.